salt/client/ssh/wrapper/grains.py

```diff
@@ -67,7 +69,35 @@ def get(key, default=''):
 
         salt '*' grains.get pkg:apache
     '''
-    return salt.utils.traverse_dict_and_list(__grains__, key, default)
+    if ordered is True:
```

Comment:
I copied this ordered section from salt.modules/grains.py, it makes no sense to me